internal/api/sms_provider/twilio.go

```diff
@@ -54,6 +54,60 @@ func NewTwilioProvider(config conf.TwilioProviderConfiguration) (SmsProvider, er
 	}, nil
 }
 
+func (t *TwilioProvider) SendMessage(phone string, message string, channel string) error {
+	switch channel {
+	case SMSProvider:
+		return t.SendSms(phone, message)
+	case WhatsappProvider:
+		return t.SendWhatsappMessage(phone, message)
+	default:
+		return fmt.Errorf("channel type %q is not supported for Twilio", channel)
+	}
+}
+
+// Send a Whatsapp message containing the OTP with Twilio's API
+func (t *TwilioProvider) SendWhatsappMessage(phone string, message string) error {
+	body := url.Values{
+		"To":      {"whatsapp:" + "+" + phone}, // twilio api requires "+" extension to be included
+		"Channel": {"whatsapp"},
+		"From":    {"whatsapp:" + t.Config.MessageServiceSid},
+		"Body":    {message},
+	}
```

Comment:
Missed this in the earlier review but since the main difference between `SendWhatsappMessage` and `SendSms` is the values in the request body, i think we can reduce code duplication by having a single `SendSms` function here. 

We can then have a switch-statement based on the channel passed in as an argument and decide how the request body should look like. 

Ofc, if you think it's not necessary at this stage, we can opt to clean this up at a later point in time

Reply:
I'll add a todo and clean it up once we're certain it's stable - thanks for flagging this though!